src/helm/proxy/clients/huggingface_model_registry.py

```diff
@@ -71,36 +76,54 @@ def from_string(raw: str) -> "HuggingFaceModelConfig":
             namespace=match.group("namespace"), model_name=model_name, revision=match.group("revision")
         )
 
+    @staticmethod
+    def from_path(path: str) -> "HuggingFaceModelConfig":
```

Comment:
Thought: Right now we use the presence of `path` to indicate if a model is local.

I almost feel like we should rename `HuggingFaceModelConfig` -> `HuggingFaceHubModelConfig` and make a separate `HuggingFaceLocalModelConfig` that only has `name` and `path`.

It's too bad a change to make in this PR, but maybe something to consider for a future PR.

Reasoning:

- Namespace doesn't make sense for local models (see thread below)
- Revision doesn't make sense for local models (a revision is equivalent to git commit, and there is no git for local models)